docs/7.x/changelog.md

```diff
@@ -570,6 +570,20 @@ to learn how to gain insight into how the cluster status changes over time.
 
 ## 6.1 Releases
 
+### 6.1.38 LTS (September 11th, 2020)
+
+#### Improvements
+
+* Increase default `LimitNOFile` parameter within planet's systemd to `655350` ([#2092](https://github.com/gravitational/gravity/pull/2092), [planet#743](https://github.com/gravitational/planet/pull/743)).
+* Upgrade Helm to `v2.15.2` ([#2118](https://github.com/gravitational/gravity/pull/2118), [planet#747](https://github.com/gravitational/planet/pull/747)).
+* Shrink operation resiliency improvements ([#2096](https://github.com/gravitational/gravity/issues/2096)).
```

Comment:
I'd mention [this](https://github.com/gravitational/planet/pull/745) planet update that address https://gravitational.zendesk.com/hc/en-us/requests/1880